src/components/Docs/components/GovUpgradeProposals.tsx

```diff
@@ -69,19 +95,25 @@ export const GovUpgradeProposals = () => {
         if (activeTab.networkType === "mainnet") setMainnetProposals(softwareUpgradeProposals);
         if (activeTab.networkType === "testnet") setTestnetProposals(softwareUpgradeProposals);
       } catch (error) {
-        console.error("Error fetching data:", error);
+        console.error("Error fetching proposals:", error);
       } finally {
         setIsLoading(false);
       }
     };
 
-    fetchData();
+    fetchProposals();
   }, [activeTab.networkType]);
 
   const proposals = useMemo(() => {
     return activeTab.networkType === "mainnet" ? mainnetProposals : testnetProposals;
   }, [activeTab.networkType, mainnetProposals, testnetProposals]);
 
+  const calculateUpgradeTime = (upgradeHeight: number) => {
+    const blocksRemaining = upgradeHeight - latestBlock;
+    const secondsRemaining = blocksRemaining * 5.5;
```

Comment:
I believe we're closer to 6s per block than 5.5s, it may lead to a bit more accurate times
```suggestion
    const secondsRemaining = blocksRemaining * 6;
```